### PR TITLE
Added Cookies popup 

### LINKED ISSUE #14: Cookies usage info for user
# Description
It is required from websites to inform users about the usage of their data. We need to add a little popup on landing page informing about the usage of the cookies. (Upfront: **Yes, we will use cookies**)

## Reference links
[Good Reference popup](https://schronisko-gorzow.pl/)
- not invasive
- simple

[What are cookies and how are they used](https://allaboutcookies.org/what-is-a-cookie)
- nicely described the purpose of cookies
- contains info how to disable/delete cookies

### PR DESCRIPTION
**Pull request by Sybirsoldat**
### Description

I added popup informing user that our website uses cookies. After agreeing to cookies, this information is stored in browser storage.

### How To Test

1. Start Server
2. Open Landing page
3. Open dev tools, application->storage->local storage
4. if pop up is visible, then click on the `I Agree` button
5. check local storage in dev tools, there should be variable `cookies` set to 1
6. refresh the page and check if the pop up shows up (it shouldn't)

### CODE REVIEW

**Review by xEdziu — CHANGES_REQUESTED** [review]
# Problem description

On bigger screen resolutions, popup is to high, it should be "glued" to the bottom of the screen. You need to find a solution to this 
image

File: index.php [hunk 1]
```diff
@@ -20,8 +20,28 @@
     </main>
     <footer class="andika-regular">
         <h2>Authors:</h2>
-        <p>Wiktor Siepka, Mateusz Andrzejewski</p>
+        <p>Wiktor Siepka, Mateusz Andrzejewski, Szymon Nowicki</p>
         <p><a class="personalLink" href="https://github.com/xEdziu">Adrian Goral</a>, Mateusz Zubrzycki</p>
     </footer>
+    <div class="popup" id="cookies">
+    <span class="pinfo andika-regular" >RPG Handy Helper uses cookie files in order to work properly [&nbsp
```
Comment (xEdziu) [inline index.php]: After second thought, I would slightly change the text here to:
RPG Handy Helper uses cookies in order to work properly

**Comment by Sybirsoldat** [comment]
> # Problem description
> On bigger screen resolutions, popup is to high, it should be "glued" to the bottom of the screen. You need to find a solution to this image

This is not problem created by me, it comes from initial page design, it should be addresed with @Myjem

**Comment by xEdziu** [comment]
> > # Problem description
> > On bigger screen resolutions, popup is to high, it should be "glued" to the bottom of the screen. You need to find a solution to this image
> 
> This is not problem created by me, it comes from initial page design, it should be addresed with @Myjem

I see, then for me it is fine

File: index.php (hunk 1 above)
Comment (Sybirsoldat) [reply index.php]: I did it

**Review by xEdziu — APPROVED** [review]
Great, much better now

**Review by Myjem — APPROVED** [review]

**Pull request merged**